docs/media-airplay-button.md

```diff
@@ -0,0 +1,28 @@
+# `<media-seek-backward-button/>`
+
+Button to jump back 30 seconds in the media.
+
+- [Source](../src/js/media-airplay-button.js)
+- [Example](https://media-chrome.mux.dev/examples/control-elements/media-airplay-button.html) ([Example Source](../examples/control-elements/media-airplay-button.html))
+
+# Attributes
+
+_None_
+
+# Slots
+
+| Name      | Default Type | Description                                              |
+| --------- | ------------ | -------------------------------------------------------- |
+| `airplay` | `svg`        | The element shown for the AirPlay button's display |
```

Comment:
No change needed: Seeing the slot name in this context makes me thinks these button slots could use more explicit names. But that might be overthinking it.